lib-ee/emqx_ee_connector/src/emqx_ee_connector_rocketmq.erl

```diff
@@ -75,7 +75,7 @@ add_default_fn(OrigFn, Default) ->
         (Field) -> OrigFn(Field)
     end.
 
-server() ->
+servers() ->
     Meta = #{desc => ?DESC("server")},
```

Comment:
```suggestion
    Meta = #{desc => ?DESC("server"), aliases => [server]},
```

Reply:
This is a new feature that hasn't been released yet.  So I think there should be no need for an alias